airbyte-integrations/connectors/destination-databend/destination_databend/client.py

```diff
@@ -6,15 +6,17 @@
 
 
 class DatabendClient:
-    def __init__(self, host: str, port: int, database: str, table: str, username: str, password: str = None):
+    def __init__(self, host: str, port: int, database: str, table: str, username: str, ssl: bool, password: str = None):
         self.host = host
         self.port = port
         self.database = database
         self.table = table
         self.username = username
         self.password = password
+        self.ssl = ssl
```

Comment:
```suggestion
        self.ssl = ssl or False
```

This way won't break existing connections.

Reply:
I think it has a default value set in spec, which is also set to true.